lib/simple_form/inputs/citext_input.rb

```diff
@@ -0,0 +1,19 @@
+module SimpleForm
```

Comment:
Since we are using Ruby 2.4, could you add the special comment: `# frozen_string_literal: true`?

Reply:
@feliperenan fixd